mmv1/third_party/terraform/utils/header_transport.go

```diff
@@ -20,6 +21,10 @@ func newTransportWithHeaders(baseTransit http.RoundTripper) headerTransportLayer
 	if requestReason := os.Getenv("CLOUDSDK_CORE_REQUEST_REASON"); requestReason != "" {
 		headers.Set("X-Goog-Request-Reason", requestReason)
 	}
+	// opt in extension for adding to the User-Agent header
```

Comment:
Can we set this in `config.go` instead so all of the user agent stuff is in one place?

Reply:
done